test/extended/cli/explain.go

```diff
@@ -329,7 +329,6 @@ var _ = g.Describe("[cli] oc explain", func() {
 	defer g.GinkgoRecover()
 
 	oc := exutil.NewCLI("oc-explain", exutil.KubeConfigPath())
-	crdClient := apiextensionsclientset.NewForConfigOrDie(oc.AdminConfig())
```

Comment:
`oc.AdminConfig` loads a config file from a disk - this caused panic when the config file wasn't present. Having a config is not always required for example when passing `dry-run` flag.

I glanced over `test/extended` just to see if the other tests don't load `AdminConfig` too early.